src/app/busca-ativa/citopatologico/Cito.tsx

```
@@ -1,79 +1,97 @@
+	const [showSnackBar, setShowSnackBar] = useState({
+		open: false,
+	});
+	const [filtrosAplicados, setFiltrosAplicados] = useState(false);
+	const [activeTabIndex, setActiveTabIndex] = useState(0);
+	const [activeTitleTabIndex, setActiveTitleTabIndex] = useState(0);
+	const [voltarGatilho, setVoltarGatilho] = useState(0);
+	const [tabelaData, setTabelaData] = useState([]);
+	const router = useRouter();
+	const path = usePathname();
+	const Voltar = () => window.history.go(voltarGatilho * -2);
+	useEffect(() => {
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: path

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)